core/storage/user/gae_models.py

```diff
@@ -1421,12 +1499,17 @@ def get_deletion_policy():
         """
         return base_models.DELETION_POLICY.DELETE
 
+    @staticmethod
+    def get_export_method():
+        """Model is exported as a multiple unshared instances."""
+        return base_models.EXPORT_METHOD.MULTIPLE_UNSHARED_INSTANCES
+
     @classmethod
     def get_export_policy(cls):
         """Model contains user data."""
         return dict(super(cls, cls).get_export_policy(), **{
             'user_id': base_models.EXPORT_POLICY.NOT_APPLICABLE,
-            'collection_id': base_models.EXPORT_POLICY.EXPORTED,
+            'collection_id': base_models.EXPORT_POLICY.NOT_APPLICABLE,
```

Comment:
Why change this?

Also, it does seem to be included below, in export_data(), as a key. So "not applicable" seems to be signalling the wrong thing?

Reply:
Same as above. If we want to change this, we can do that, I'll just have to add another test.

Reply:
Discussed offline, we decided to add another enum value EXPORTED_AS_DATA_IDENTIFIER for these cases.

Reply:
Discussed again to change this to EXPORTED_AS_MODEL_TAKEOUT_ID